src/Metrics/DefaultMetricsSender.php

```diff
@@ -33,6 +34,11 @@ public function sendMetrics(MetricsBucket $bucket): void
         foreach ($this->configuration->getHeaders() as $name => $value) {
             $request = $request->withHeader($name, $value);
         }
-        $this->httpClient->sendRequest($request);
+
+        try {
+            $this->httpClient->sendRequest($request);
+        } catch (ClientExceptionInterface) {
+            // ignore the error
```

Comment:
This looks intentionally ignored but do we not want to log this? Totally okay with the answer being no here

Reply:
It doesn't currently log anything as there's no logging set up, not sure how to log this, the only thing I can come up with is [`error_log()`](https://www.php.net/manual/en/function.error-log.php). The SDK would probably need some logger support in general, not sure it's worth creating a partial support here.

Reply:
No sounds super sensible to me!